torch/_inductor/codegen/cpp.py

```diff
@@ -2486,7 +2486,7 @@ def reduction_acc_type_vec(self, reduction_type, dtype):
 
     def welford_weight_reciprocal_vec(self, dtype, num_threads=None):
         vec_num_range_thread = (
-            CeilDiv(self.weight_recp_vec_range, num_threads)
+            CeilDiv(self.weight_recp_vec_range, sympy.sympify(num_threads))
```

Comment:
I should just make CeilDiv sympify its arguments, __new__ style custom functions don't auto sympify

Reply:
Leave this in the tracking issue.

Reply:
Just going to fix it right now